roles/docker/vars/ubuntu.yml

```diff
@@ -1,15 +1,13 @@
 ---
 docker_kernel_min_version: '3.10'
+docker_package_arch: 'amd64'
 
-# https://apt.dockerproject.org/repo/dists/ubuntu-xenial/main/filelist
+# https://download.docker.com/linux/ubuntu/dists/trusty/stable/
 docker_versioned_pkg:
-  'latest': docker-engine
-  '1.11': docker-engine=1.11.1-0~{{ ansible_distribution_release|lower }}
-  '1.12': docker-engine=1.12.6-0~ubuntu-{{ ansible_distribution_release|lower }}
-  '1.13': docker-engine=1.13.1-0~ubuntu-{{ ansible_distribution_release|lower }}
-  '17.03': docker-engine=17.03.1~ce-0~ubuntu-{{ ansible_distribution_release|lower }}
```

Comment:
no need to remove the previous versions as it may break existing users/configurations

Reply:
These don't appear to be present in the new repo however - so we may need to include both apt repos to do so.